docs/horizen_eon/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -3,7 +3,7 @@ title: How to Stake on EON
 ---
 
 # How to Stake on EON
-A forger is required to run a node. See [How to Locally Build and Deploy](../04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md) for information on creating and running a node.  
+A forger is required to run a node. See [How to Locally Build and Deploy](../04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md) for information on [creating](./04-forger_node.md) and running a node.  
```

Comment:
```suggestion
A forger is required to run a node. See [How to Locally Build and Deploy](../04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md) and [Forger Node Requirements](./04-forger_node.md) for information on creating and running a node.  
```

Reply:
I just fixed this...not sure why it showed up again.